tests/detect-ldap-result/README.md

```diff
@@ -0,0 +1,5 @@
+Test ldap.responses.result_code and ldap.responses.error_message keywords.
+
+PCAP created with flowsynth.py
```

Comment:
This is ok but I wonder if we could find "real" pcaps (not synthetic ones, but ones with a real LDAP client talking to a real LDAP server)